client/cypress/integration/dashboard/dashboard_spec.js

```diff
@@ -402,26 +376,28 @@ describe('Dashboard', () => {
         });
 
         it('stays put when dragged under snap threshold', () => {
-          resizeBy(cy.get('@textboxEl'), 0, 10).then((delta) => {
-            expect(delta.height).to.eq(0);
-          });
+          resizeBy(cy.get('@textboxEl'), 0, 10)
+            .then(() => cy.get('@textboxEl'))
```

Comment:
I think this being necessary is a sign that passing a `cy.get()` as an argument is an anti-pattern (similar to [this one](https://docs.cypress.io/guides/references/best-practices.html#Assigning-Return-Values)). We should rather use it as a Cypress command or pass something else as argument.

For now I think it's ok to leave as is, but let's keep in mind this to when we discover a better way to treat those helper commands.

Reply:
I worked on making `dragBy` and `resizeBy` into commands but it didn't go smooth so I'm leaving as is to keep moving forward.